boards/native/nrf_bsim/nrf54l15bsim_nrf54l15_cpuapp.dts

```diff
@@ -103,3 +103,8 @@
 &clock {
 	status = "okay";
 };
+
+&rng {
+	status = "okay";
+	compatible = "zephyr,native-posix-rng";
+};
```

Comment:
Isnt it already set to "okay" here: https://github.com/zephyrproject-rtos/zephyr/pull/77749/files#diff-321c52f04112392bcc5d28561a6f0ee142fae8362e0d214adf0aa3839f327157R69 ?

Reply:
you are fully right..